File: Phyton/Ex37.py
N = int(input("Digite um número positivo, menor que 20: "))
while(N <= 0)or(N>20):
 print("Não pode número negativo!")
 N = int(input("Digite um outro número positivo, menor que 20: "))

numeros = []
m=1

for i in range(0, N, 1):
 x = int(input('Digite um numero: '))
 numeros.append(x)

while(m==1):
 en = int(input('Digite um numero para encontrar: '))
 for i in range(0, N, 1):
   if(en==numeros[i]):
    print(f'Posição: {i}')
    contagem=0

 if(contagem==1):
     print(f'Valor não encontrado!') 
     m = int(input('Digite 1 para procurar por um outro valor, ou 0 para não procurar:'))

File: C/Ex37.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    printf("Digite a quantia, sendo maior que 0 e menor que 20: ");
    int numeros[20], numero, quantia, contagem=0, repete;

    scanf("%i",&quantia);
    if((quantia<=0)||(quantia>20))
    {
        while((quantia)<=0||(quantia>20))
        {
            printf("Erro, digite um numero maior que 0 e menor que 20: ");
            scanf("%i",&quantia);
        }
    }
   
    for (int i=0; i<quantia; i++)
    {
    scanf("%i",&numeros[i]);
    }
    while(repete==0)
    {
     printf("Digite um numero para o programa encontrar a posicao: ");
     scanf("%i",&numero);   

     for (int i=0; i<quantia; i++)
      {  
       if(numero==numeros[i])
       {
            printf("Posicao: %i\n",i);
            contagem=1;
       }
     }
     
     
    if(contagem==0)
    {
        printf("Valor nao encontrado!");
        printf("Digite 0 para repitir ou 1 para não repitir o número para encontrar: ");
     scanf("%i",&repete);
    }
     
    }
    printf("\n Finalizando o programa...");

    return 0;
}